common/app/routes/challenges/components/Classic-Modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { PropTypes } from 'react';
```

Comment:
It seems like you are creating a generic modal here, but we already have that with bootstrap. Just use the bootstrap modal directly, no need to make this a generic component.